scss/_patterns_code-numbered.scss

```diff
@@ -4,18 +4,44 @@
     background: $color-light;
     color: $color-dark;
     counter-reset: line-numbering;
-    padding: $sp-medium;
+    padding: 0 $sp-medium 0 0;
 
-    .code-line::before {
-      color: $color-mid-dark;
-      content: counter(line-numbering);
-      counter-increment: line-numbering;
-      display: inline-block;
-      padding-right: $sp-medium; // space after numbers
-      pointer-events: none;
-      text-align: right;
-      user-select: none;
-      width: $sp-xxx-large;
+    code {
+      padding: 0 0 0;
+      position: relative;
```

Comment:
It doesn't look like these styles change anything and we should not target element in patterns.